lib/codit/society/basic.py

```diff
@@ -7,6 +7,7 @@ def __init__(self, episodes_per_day=None, encounter_size=None, prob_unnecessary_
         if not prob_unnecessary_worry:
             prob_unnecessary_worry = self.cfg.PROB_NON_C19_SYMPTOMS_PER_DAY
         self.episodes_per_day = episodes_per_day or self.cfg.SIMULATOR_PERIODS_PER_DAY
+        assert type(self.episodes_per_day) == int
```

Comment:
Is ```assert``` the correct thing here?  assert is for debugging and testing, ie: for finding errors in your code, not for checking errors in the parameters or config that is passed to your code. Maybe you should raise a ValueError here?